### ISSUE TITLE
Update release notes for photospline (Trac #1974)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1966]
Migrated from https://code.icecube.wisc.edu/ticket/1974
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:13:35", 
    "description": "", 
    "reporter": "juancarlos", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067215093672", 
    "component": "combo simulation", 
    "summary": "Update release notes for photospline", 
    "priority": "blocker", 
    "keywords": "photospline", 
    "time": "2017-03-23T18:48:00", 
    "milestone": "", 
    "owner": "jvansanten", 
    "type": "task"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20170324T19:15:38: 
* jvansanten commented:

> In changeset 154266:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="154266"
> Update release notes; closes #1966
> ```


* jvansanten changed status from "new" to "closed"
* jvansanten changed resolution from "" to "fixed"

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:13:35: 
* olivas commented:

> Milestone IceSim 5.1 deleted


* olivas changed milestone from "IceSim 5.1" to ""
